plugins/modules/ssm_parameter.py

```diff
@@ -387,7 +493,27 @@ def create_update_parameter(client, module):
                     (changed, response) = update_parameter(client, module, **args)
         if changed:
             _wait_updated(client, module, module.params.get('name'), original_version)
+            # import time
+            # time.sleep(300)
+
+        # Handle tag updates for existing parameters
+        if (module.params.get('overwrite_value') != 'never'):
```

Comment:
```suggestion
        if module.params.get('overwrite_value') != 'never':
```

Reply:
Thanks Dennis, Done. I also removed the useless commented out code on line 497.